admin/services/data-access/sql.pool.service.js

```diff
@@ -41,10 +41,10 @@ sqlPoolService.init = () => {
   })
 }
 
-  /**
-   * Get a connection from the pool.
-   * @return {Promise}
-   */
+/**
+ * Get a connection from the pool.
+ * @return {Promise}
```

Comment:
Same here

Reply:
getConnection returns a promise, see the full code

Reply:
it certainly does resolve `resolve(connection)`

Reply:
ok github diff didn't help me on seeing this clearly.